pull_request_template.md

```diff
@@ -0,0 +1,27 @@
+# Description
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+
+# How Has This Been Tested?
+
+Please describe the tests that you ran to verify your changes. Please also note any relevant details for your test configuration.
+
+
+
```

Comment:
The answers to this section are likely to be the same. I wonder if we can add a list to the boilerplate so that people can delete some items and just keep the ones they need